### PR TITLE
Implemented Plus/Minus/Times for Array1

### PR DESCRIPTION
**Pull request by qindazhu**

### CODE REVIEW

File: k2/csrc/array_ops_test.cu [hunk 1]
```diff
@@ -1342,6 +1337,144 @@ TEST(OpsTest, InvertMonotonicDecreasingTest) {
   }
 }
 
+template <typename T>
+void ArrayPlusTest() {
+  ContextPtr cpu = GetCpuContext();  // will be used to copy data
+  for (auto &context : {GetCpuContext()}) {
```
Comment (csukuangfj) [inline k2/csrc/array_ops_test.cu:1343]: Shall we also enable cuda test?

Comment (qindazhu) [reply k2/csrc/array_ops_test.cu:1343]: Oh, sorry, forget to add it back after testing.

Comment (qindazhu) [reply k2/csrc/array_ops_test.cu]: Fixed, thanks!

**Comment by danpovey** [comment]
Cool.. pls merge..

**Pull request merged**